src/pages/create-test/FieldsComponents/SelectorField/SwitchSelector.tsx

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+import SwitchSelector from 'react-switch-selector';
+import { palette } from '../../../../styles/theme/commonDefaultTheme';
+import { SelectorType } from '../../../../types/SelectorType';
+
+export const SwitchSelectors = ({ onPress, type, value }: SelectorType) => {
+  const SelectorsData = useMemo(
+    () =>
+      // eslint-disable-next-line no-nested-ternary
```

Comment:
better refactor to if () return, rather then adding eslint-disabe

Reply:
refactor this part of code using if()